File: 41-vetor-simples/foo2.js
var assert = require('assert');


var i, vetor = [];

for(i = 1; i < 6; i++ ){
    vetor.push(i)
}

//
// Testes
//
try {

    assert.equal(1, vetor[0]);
    assert.equal(2, vetor[1]);
    assert.equal(3, vetor[2]);
    assert.equal(4, vetor[3]);
    assert.equal(5, vetor[4]);

} catch(e) {
    console.log(e);
}

File: 45-vetor-valores-repetidos/foo2.py
# -*- coding: utf-8 -*-

#
# Sua lógica...
#
def removerDuplicados(duplicado):
    _lis = []

    for x in duplicado:
        if x not in _lis:
            _lis.append(x)

    return _lis

def haDuplicados(lista):
    if len(lista) != len(removerDuplicados(lista)):
        return True
    else:
        return False





#
# Testes
#
assert haDuplicados([100, 200, 300, 300, 500])
assert not haDuplicados([100, 200, 300, 400, 500])
